powerlibs/django/restless/contrib/endpoints/tool.py

```diff
@@ -0,0 +1,29 @@
+class SieveListFieldDeleted:
+
+    @staticmethod
+    def delete_field_list(data):
+        if data is not None:
```

Comment:
I believe you could reimplement the `serialize` method or something like that. I recommend you to research that in `powerlibs-django-restless`, file `modelviews.py` (if I remember right).